src/components/Answer.vue

```diff
@@ -27,7 +27,9 @@ export default Vue.extend({
   name: 'Answer',
   data: () => ({
     displayTitles: global.piafAgentConfig.DISPLAY_TITLES,
-    displayProbability: global.piafAgentConfig.DISPLAY_PROBABILITIES
+    displayProbability: global.piafAgentConfig.DISPLAY_PROBABILITIES,
+    displayTag: global.piafAgentConfig.DISPLAY_TAG,
+    tagname: global.piafAgentConfig.TAGNAME
```

Comment:
I think you can get rid of the DISPLAY_TAG boolean, by having it deduced from the TAGNAME
```suggestion
    displayTag: global.piafAgentConfig.TAGNAME !== "",
    tagname: global.piafAgentConfig.TAGNAME
```

and, in the loadConfig, `TAGNAME: configFromJson.TAGNAME ?? '',`

Reply:
true

Reply:
should be fixed now